nevergrad/instrumentation/multivariables.py

```diff
@@ -188,6 +188,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         super().__init__()
         self.keywords: Tuple[Optional[str], ...] = ()
         self.probably_noisy = False  # True if for some reason we conjecture that the objective function is noisy.
+        self.scrambled = False  # True if there is no metric on the domain.
```

Comment:
what is this?

Reply:
This is true when the metric on the input domain is pointless.
Useful for some applications in which we choose seeds.